deep_translator/deepl.py

```diff
@@ -67,6 +67,9 @@ def translate_batch(self, batch, **kwargs):
         """
         return [self.translate(text, **kwargs) for text in batch]
 
+    def get_supported_languages(**kwargs):
```

Comment:
@sterbon Did you tested this? You need to mark this as a static function

Reply:
I actually did test it with the script provided in #69 

The output is printed as shown.

```
<class 'deep_translator.deepl.DeepL'>
['bulgarian', 'czech', 'danish', 'german', 'greek', 'english', 'spanish', 'estonian', 'finnish', 'french', 'hungarian', 'italian', 'japanese', 'lithuanian', 'latvian', dutch', 'polish', 'portuguese', 'romanian', 'russian', 'slovak', 'slovenian', 'swedish', 'chinese']
<class 'deep_translator.qcri.QCRI'>
['Arabic', 'English', 'Spanish']
```

Reply:
I have marked the functions as static now.

Reply:
alright thanks